services/shared/serverless.yml

```diff
@@ -0,0 +1,53 @@
+service: shd-monorepo-example # To be replaced
```

Comment:
@kai-nguyen-aligent comment from downstream PR:

I'm not sure if this file is necessary. Why don't we just merge this file and the serverless.common.yml? I have the same feeling about the package.json file in this /services/shared folder.

Reply:
@kai-nguyen-aligent the common.yml doesnt actually 'build' anything within AWS. It is purely there to define variables that are used across all services - e.g. the basic provider details.

The purpose of the shared 'service' is to build out tables/buckets etc that are not 'owned' by a particular service. Example being a product table that maps ids from one system to another that would be used by an order, inventory, and product service.

I guess this comes to the question of should we declare things like this as a 'shared' resource? Or should we declare it in the most appropriate service - with the example above that would likely be the product service.

@kai-nguyen-aligent @tvhees - This might be a wider discussion that we can have about how we want to declare items that are used across multiple services